File: Java/231A-Team.java
import java.util.Scanner;

public class Main{

     public static void main(String []args){
        int n, sum = 0;
        boolean a, b, c;
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();
        
        for(int i = 0; i < n; i++){
            a = (sc.nextInt() != 0);
            b = (sc.nextInt() != 0);
            c = (sc.nextInt() != 0);
            if((a && b) || (b && c) || (c && a)){
                sum++;
            }
        }
        System.out.println(sum);
     }
}

File: C++/231A-Team.cpp
#include <iostream>

int main() {
    int n, sum = 0;
    bool a, b, c;
    std::cin >> n;
    
    for(int i = 0; i < n; i++){
        std::cin >> a >> b >> c;
        if((a && b) || (b && c) || (c && a)){
            sum++;
        }
    }
    
    std::cout << sum;
    return 0;
}
